### PR TITLE
[circle-quantizer] Propagate quantparam forward direction

### PR DESCRIPTION
**Pull request by jinevening**
This propagates quantparam in a forward direction.

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

### CODE REVIEW

**Review by seanshpark — DISMISSED** [review]
LGTM

File: compiler/luci/pass/src/PropagateQuantParamPass.cpp [hunk 1]
```diff
@@ -70,7 +70,7 @@ struct PropagateQuantParam final : public luci::CircleNodeMutableVisitor<bool>
       return false;
 
     auto input_node = loco::must_cast<luci::CircleNode *>(input);
-    return copy_qparam(node, input_node);
+    return copy_qparam(input_node, node);
```
Comment (seanshpark) [inline compiler/luci/pass/src/PropagateQuantParamPass.cpp:73]: this was a bug?

Comment (jinevening) [reply compiler/luci/pass/src/PropagateQuantParamPass.cpp:73]: Yes

**Comment by seanshpark** [comment]
@nnfw-bot test nncc-debug

**Comment by seanshpark** [comment]
@nnfw-bot test nncc-release

**Comment by seanshpark** [comment]
@nnfw-bot test nncc-debug

**Comment by seanshpark** [comment]
@nnfw-bot test nncc-release"

**Comment by seanshpark** [comment]
```
07:06:03 /opt/jenkins_slave/workspace/nnfw/master/pr-nncc-release/compiler/luci/pass/src/PropagateQuantParamPass.test.cpp:101: Failure
07:06:03       Expected: 0.2
07:06:03 To be equal to: g.conv->quantparam()->scale[0]
07:06:03       Which is: 0.1
```
...

**Comment by jinevening** [comment]
Oops.. I forgot to update the test data. I've uploaded an updated test. PTAL

**Review by seanshpark — APPROVED** [review]
LGTM

**Pull request merged**